mesos-rxjava-core/src/main/java/org/apache/mesos/rx/java/MesosSchedulerClient.java

```diff
@@ -114,4 +138,75 @@ public Subscription sink(@NotNull final Observable<SinkOperation<Send>> spout) {
             .observeOn(Rx.compute())
             .subscribe(subscriber);
     }
+
+    @VisibleForTesting
+    static final class UserAgent {
+        private static final Joiner JOINER = Joiner.on(" ").skipNulls();
```

Comment:
Although, we should probably start using a linter to detect these automatically...